python-setup/python-setup-suggestions.qmd

```diff
@@ -0,0 +1,154 @@
+
+The Positron IDE can help you do both. You can use it to work with two common types of environments:
+
+* **Virtual Environments** - which are Python's [built-in environments](https://docs.python.org/3/glossary.html#term-virtual-environment). A virtual environment is a subdirectory that contains a copy or a symlink to a Python interpreter, as well as a library of packages to be used by the environment.
+* **Conda environments** - which are Python environments managed by the [conda package manager](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html).
+
+## Which environment are you using?
+
+When you open a project in Positron, Positron will show you which environment it is using at the top of your IDE. Here, Positron is using an environment contained in a folder named .venv and running the Python 3.12.2 interpreter. 
+
+![Environment button](images/button.png)
+
+To see the full filepath to the .venv folder, click on the button.
+
+![Environment dropdown](images/dropdown.png)
+
+Positron will use this environment when you run Python commands in your Terminal and Session windows. It will also use the environment if you do something that triggers a Python process, like previewing a Quarto doc or running a Shiny for Python app.
+
+Whenever you _re_open a folder in Positron, Positron will activate the environment you used the last time you worked in the folder. If this is your first time opening the folder, Positron will make an educated guess about which environment to use or ask you to select an environment. In that case, the button will display Start Interpreter.
+
+![Start interpreter](images/start-interpreter.png)
+
+## How to change your environment
+
+To change the environment, open the drop down menu and click the three dots next to the current environment's name. Positron will list the environments it detects on your machine. Select the one you want to use.
+
+![Enviornments list](images/list.png){width=300}
+
+But what if you want to create a new environment specific to your project? 
+
+## Create a new environment
+
+A good best practice is to create a new environment for each new project. This forces you to be intentional about your environment, but it also protects your other projects. If you run each project in the same global environment, then any changes you make with one project will affect other projects. Not so if each project uses its own environment.
+
+So how do you do it? To create a new environment:
+
+1. Open the Command Palette with **Command + Shift + P** (Mac) or **Control + Shift + P** (Windows and Linux)
+
+![Command Palette](images/palette.png)
+
+2. Select the **Python: Create Environment** command.
+
+3. Then select the type of environment you would like to create: Venv or Conda.
+
+![Select type](images/type.png)
+
+4. Next select the Python interpreter (venv) or Python version (conda) that you would like to use for the environment.
```

Comment:
If we make this intro article just venv, maybe add a footnote or link down to where we talk about other kinds of Pythons.

Reply:
Done. https://github.com/garrettgman/positron-guides/commit/bbd62a337aed3a758ef790db99b6712c62247015